### PR TITLE
fix api access to activate/deactive natEnabled

### PR DESCRIPTION
**Pull request by lkwg82**
this fixes the changed api access to internal syncthing

How can the translation changed?

### CODE REVIEW

**Comment by Nutomic** [comment]
Change the english string in `res/values/strings.xml`, I will then push it to Transifex. You should probably change the string's key for consistency, and delete the string from all other language folders.

**Comment by lkwg82** [comment]
Plz. delete branches on merge, thx!

File: src/main/res/values/strings.xml [hunk 1]
```diff
@@ -274,7 +274,7 @@ Please report any problems you encounter via Github.</string>
 
     <string name="local_announce_enabled">Local Discovery</string>
 
-    <string name="upnp_enabled">Enable UPnP</string>
+    <string name="nat_enabled">Enable NAT</string>
```
Comment (AudriusButkevicius) [inline src/main/res/values/strings.xml]: What does Enable NAT mean? 
NAT traversal maybe?

Comment (Nutomic) [inline src/main/res/values/strings.xml]: The web gui says "Enable NAT traversal", so I think we should go with that.

Comment (lkwg82) [inline src/main/res/values/strings.xml]: Of course, I missed it.

**Comment by Nutomic** [comment]
Thanks!

**Pull request merged**